### DISCUSSION TITLE
Error please help

### DISCUSSION THREAD

**Discussion by stelios954** [MlgmXyysd/Xiaomi-HyperOS-BootLoader-Bypass#196] [Q&A]
When i try to bypass i get PHP Fatal error:  Uncaught Error: Call to undefined function openssl_decrypt() in C:\Users\stelios954\Desktop\php\bypass.php:230
Stack trace:
#0 C:\Users\stelios954\Desktop\php\bypass.php(330): decryptData('8/8Mnp7WcRph8sb...')
#1 {main}
  thrown in C:\Users\stelios954\Desktop\php\bypass.php on line 230

Fatal error: Uncaught Error: Call to undefined function openssl_decrypt() in C:\Users\stelios954\Desktop\php\bypass.php:230
Stack trace:
#0 C:\Users\stelios954\Desktop\php\bypass.php(330): decryptData('8/8Mnp7WcRph8sb...')
#1 {main}
  thrown in C:\Users\stelios954\Desktop\php\bypass.php on line 230

I dont know hot to enable curl and openssl can someone help me

**Comment by KevinChin-TFM** [comment]
enable the ‘extension_dir = "ext" ', 'extension=curl', 'extension=openssl' in php.ini.
’
![image](https://github.com/user-attachments/assets/da9c40dd-f103-459d-8aaf-fc7becd10fb0)
